### DISCUSSION TITLE
Use Antlers to set the default value of a form field

### DISCUSSION THREAD

**Discussion by QuentinGab** [statamic/cms#6483] [Q&A] [answered]
Hi,
My question is very basic but I havn't been able to find any answer in the docs or in discussions here.

Basically, I have a form text field called "first_name" and I'm trying to set its default value to the current user first_name.
It would be perfect if I could just set the field default value to `{{ current_user:first_name }}` with `antlers : true` in cp but as you might expect it litterally output `<input value="{{ current_user:first_name }}" />` in the DOM (not parsed by antlers). 

Am I doing something wrong ? Do I even understand what `antlers : true` stands for ? So many questions

**Comment by edalzell** [comment]
Why not do just what you wrote `<input value="{{ current_user:first_name }}" />` ?

**Reply by QuentinGab** [reply]
I'm trying to set the default value of the field from the cp, so what I did is fill the default field in the bluebrint with the value `{{ current_user:first_name }}` but it is not parsed by antlers and just output it as a string.

**Reply by edalzell** [reply]
not sure that will work, but try adding `antlers: true` to the field.

**Reply by QuentinGab** [reply]
I did it too but it changes nothing, the string in default value is not parsed by antlers

**Comment by duncanmcclean** [comment, marked as answer]
I believe we did what you're explaining recently but it only worked when using the Regex parser and not the new Runtime parser. We ended up finding a workaround for our use case.

**Reply by QuentinGab** [reply]
Okay, thanks for the answer ! I will use a workaround then
